x-pack/test/security_solution_endpoint/apps/endpoint/policy_list.ts

```diff
@@ -18,6 +18,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
   const policyTestResources = getService('policyTestResources');
   const RELATIVE_DATE_FORMAT = /\d (?:seconds|minutes) ago/i;
+  const retry = getService('retry');
```

Comment:
Did you forget to delete this? (you might get a ESLint error

Reply:
yep, I sure did